service/src/test/java/com/ge/predix/controller/test/PolicyEvalWithGraphDbControllerIT.java

```diff
@@ -142,45 +142,59 @@ public void testCleanup() {
     }
 
     @Test(dataProvider = "policyEvalDataProvider")
+    public void testPolicyInvalidMediaTypeResponseStatusCheck(final Zone zone, final PolicySet testPolicySet,
+            final List<BaseResource> resourceHierarchy, final List<BaseSubject> subjectHierarchy,
+            final PolicyEvaluationRequestV1 policyEvalRequest, final Effect expectedEffect) throws Exception {
+        // Create policy set.
+
+        String uri = POLICY_SET_URL + "/" + URLEncoder.encode(testPolicySet.getName(), "UTF-8");
+        MockMvcContext putPolicySetContext = this.testUtils
+                .createWACWithCustomPUTRequestBuilder(this.wac, zone.getSubdomain(), uri);
+        putPolicySetContext.getMockMvc().perform(putPolicySetContext.getBuilder().contentType(MediaType.TEXT_HTML_VALUE)
+                .content(OBJECT_WRITER.writeValueAsString(testPolicySet))).andExpect(status().isUnsupportedMediaType());
+
+    }
+
+    @Test(dataProvider = "policyEvalDataProvider")
     public void testPolicyEvaluation(final Zone zone, final PolicySet testPolicySet,
             final List<BaseResource> resourceHierarchy, final List<BaseSubject> subjectHierarchy,
             final PolicyEvaluationRequestV1 policyEvalRequest, final Effect expectedEffect) throws Exception {
         // Create policy set.
 
         String uri = POLICY_SET_URL + "/" + URLEncoder.encode(testPolicySet.getName(), "UTF-8");
-        MockMvcContext putPolicySetContext = this.testUtils.createWACWithCustomPUTRequestBuilder(this.wac,
-                zone.getSubdomain(), uri);
-        putPolicySetContext.getMockMvc().perform(putPolicySetContext.getBuilder()
-                .contentType(MediaType.APPLICATION_JSON).content(OBJECT_WRITER.writeValueAsString(testPolicySet)))
-                .andExpect(status().isCreated());
+        MockMvcContext putPolicySetContext = this.testUtils
```

Comment:
All changes below seems to be format related and should be avoided.  Please leave only changes that are applicable for this defect.

Reply:
Fixed